devel/cpputest/Portfile

```diff
@@ -4,18 +4,26 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-categories          devel
-platforms           darwin macosx
+github.setup        cpputest cpputest 4.0 v
```

Comment:
Maybe add `github.tarball_from archive` to use `archive` links instead of Github's `tarball` links... (doing that will change the checksum).

Reply:
Added! See https://github.com/macports/macports-ports/pull/12299/files#diff-6408f455932017c9cdb8d6d5c6b007900507b259caac15dded44debae51012bdR13.

Reply:
Remember when I said doing that would update the checksum.

You need to do a `port clean -f --all cpputest` then a `port bump cpputest` to update to the new checksum after setting `github.tarball_from` to `archive`.

The correct updated checksums should be:
```
checksums           rmd160  3f59d987202a38a18c6a749c5d5c8df9b0e99cb5 \
                    sha256  0b66d20661f232d2a6af124c4455c8ccc9a4ce72d29f6ad4877eb385faaf5108 \
                    size    762773
```

Reply:
Thanks, and updated! (I missed the `--all` to `port clean`.)

Reply:
Thanks @cardi